--- icu/source/configure	2013-05-23 20:06:32.000000000 +0000
+++ icu/source/configure.patched	2018-02-21 23:35:25.934967613 +0000
@@ -7217,14 +7217,7 @@
 { $as_echo "$as_me:${as_lineno-$LINENO}: result: $msg" >&5
 $as_echo "$msg" >&6; }
 
-if test "$ICULIBSUFFIX" != ""
-then
-    U_HAVE_LIB_SUFFIX=1
-    ICULIBSUFFIXCNAME=`echo _$ICULIBSUFFIX | sed 's/^A-Za-z0-9_/_/g'`
-    UCONFIG_CPPFLAGS="${UCONFIG_CPPFLAGS} -DU_HAVE_LIB_SUFFIX=1 -DU_LIB_SUFFIX_C_NAME=${ICULIBSUFFIXCNAME} "
-else
-    U_HAVE_LIB_SUFFIX=0
-fi
+U_HAVE_LIB_SUFFIX=0
 
 
 
